shoop/admin/static_src/base/js/jquery.form-submission-attributes.polyfill.js

```diff
@@ -0,0 +1,47 @@
+(function($) {
+    "use strict";
+
+    var allowedButtons = 'input[type="submit"], input[type="image"], button:not([type]), button[type="submit"]';
+    var formAttributes = ['action', 'method', 'enctype', 'target', 'novalidate'];
+
+    $.fn.formSubmissionAttributes = function () {
+        //based on https://github.com/mattberkowitz/Form-Submission-Attributes-Polyfill
+
+        this.each(function () {
+            var $form = $(this);
+            var $inputs = $form.find(allowedButtons);
+
+
+            if ($form.attr('id')) {
```

Comment:
Hoist `$form.attr('id')` into a variable? (The expression is used thrice in the function.)